test/acsengine.deploy

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env groovy
```

Comment:
Why is this a new file rather than modifying the existing pipeline? Or, why not remove the old one if this is the way forward?